Platform/TigerlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -1213,8 +1214,10 @@ TccModePostMemConfig (
          FspsUpd->FspsConfig.CpuPcieRpL1Substates[Index] = PolicyConfig->CpuPcieRpL1;
       }
       mTccRtd3Support    = PolicyConfig->Dstates;
-      mTccLowPowerS0Idle = PolicyConfig->Sstates;
       mTccDsoTuning      = TRUE;
+
+      if (FeaturesCfgData != NULL)
+        FeaturesCfgData->Features.S0ix = PolicyConfig->Sstates;
     }

```

Comment:
why use PolicyConfig->Sstates to update FeaturesCfgData->Features.S0ix? 
this config data should not be updated.

Reply:
because previously Sstates is used to update mTccLowPowerS0Idle, which is s0ix flag also.
Hence remove mTccLowPowerS0Idle for redundancy.

Reply:
Just confirmed with the TCC DSO document, Sstates is indeed S0ix state, yes the Sstates naming is a bit confusing haha

Reply:
OK.